roles/common/tasks/tfenv.yml

```
@@ -0,0 +1,18 @@
+---
+
+- name: Check if tfenv is installed
+  stat:
+    path: "/home/{{ user }}/.tfenv/bin/terraform"
+  register: tfenv_existence
+
+- name: Download tfenv
+  git:
+    repo: 'https://github.com/tfutils/tfenv.git'
+    dest: "/home/{{ user }}/.tfenv"
+    version: master
```

Add the option as in the oh-my-zsh module to drop the state module block
Maybe you want to stick to a specific version use git tags ;)

```suggestion
    version: master
    update: no
```

---

I will update this with a tag

---

done